ledger_secure_sdk_sys/build.rs

```diff
@@ -330,8 +330,8 @@ impl SDKBuilder {
     }
 
     pub fn bolos_sdk(&mut self) -> Result<(), SDKBuildError> {
-        println!("cargo:rerun-if-env-changed=LEDGER_SDK_PATH");
-        let sdk_path = match env::var("LEDGER_SDK_PATH") {
+        println!("cargo:rerun-if-env-changed=LEDGER_C_SDK_PATH");
+        let sdk_path = match env::var("LEDGER_C_SDK_PATH") {
```

Comment:
This is used in deployment script, so if not necessary, would be better not to change it.

Reply:
Ok